README.md

```diff
@@ -887,6 +894,77 @@ gpg> save
 
 By default, the last identity added will be the primary user ID - use `primary` to change that.
 
+# Create keys with `--batch` and `--quick-add-keys`
+
+To remove some complexity from the process, we will show an alternate procedure to generate the keys using template files and the `--batch` parameter. For futher details, full GNUPG documentation can be found [at this link](https://www.gnupg.org/documentation/manuals/gnupg/Unattended-GPG-key-generation.html). This procedure will have just the same result as described above.
+
+For your convenience you can start from this RSA4096 key template: [gen-params-rsa4096](contrib/gen-params-rsa4096). If you're using GnuPG v2.1.7 or newer we strongly recommend generating ED25519 keys ([gen-params-ed25519](contrib/gen-params-ed25519), the procedure is the same). These templates will not set the master key to expire - see [Note #3](#notes).
```

Comment:
> If you're using GnuPG v2.1.7 or newer we strongly recommend generating ED25519 keys

May I ask you what is the reason for that recommendation?

Reply:
[Here's](https://blog.trailofbits.com/2019/07/08/fuck-rsa/) a decent overview.

Reply:
thank you for the link!